### DISCUSSION TITLE
5800X and 110G ramdisk benchmark

### DISCUSSION THREAD

**Discussion by vvavepacket** [madMAx43v3r/chia-plotter#557] [General]
Does anyone have any plot times for a 5800X and 110G ramdisk?

I'm getting 20 mins plot time on a 5950X.. How about for 5800X. How much do we expect improvement?

**Comment by thekysoft** [comment]
> Does anyone have any plot times for a 5800X and 110G ramdisk?
> 
> 
> 
> I'm getting 20 mins plot time on a 5950X.. How about for 5800X. How much do we expect improvement?

Hi, you plotting with nvme with temp drive and RAM disk with temp2 ?

**Comment by vvavepacket** [comment]
Yes i am

**Reply by thekysoft** [reply]
So you are running the ram with 128gb? Then how much ram size you allocate for plotting? Any recommended software to configure the RAM disk?

**Comment by cyperbg** [comment]
> Does anyone have any plot times for a 5800X and 110G ramdisk?
> 
> I'm getting 20 mins plot time on a 5950X.. How about for 5800X. How much do we expect improvement?

Can you share your settings for 5950x for 20 minutes?
